.github/dependabot.yml

```diff
@@ -12,6 +12,6 @@ updates:
       interval: "daily"
   # Requirements to run style checks
   - package-ecosystem: "pip"
-    directory: "/.github/workflows/style"
+    directory: "/.github/workflows/requirements"
```

Comment:
Just a note that we might be relying on an undocumented feature: https://github.com/dependabot/dependabot-core/issues/3940#issuecomment-1227005267

Reply:
Hoping they add official documentation soon. I learned about this functionality from @adamjstewart.

Reply:
@alalazo are you worried about merging this without the feature being documented?

Reply:
There's a conflict to be fixed. Otherwise this LGTM. The PR should probably mention reorganizing the placement of a few files, since it's 99% of what the PR does :wink: